File: 백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.py
C = int(input())

for tc in range(1,C+1):
    a = list(map(int, input().split(' ')))
    sum = 0
    cnt = 0
    for i in range(len(a)):
        if i == 0:
            continue
        
        else:
            sum += a[i]

    for i in range(len(a)):
        if i == 0:
            continue
        
        else:
            if a[i] > (sum/a[0]):
                cnt += 1
    
    print(f'{(cnt/a[0])*100:.3f}%')

File: 백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int C = Integer.parseInt(br.readLine());

        for(int tc = 1; tc < C+1; tc++) {
            StringTokenizer st = new StringTokenizer(br.readLine());
            int N = Integer.parseInt(st.nextToken());
            int[] arr = new int[N];
            double sum = 0;
            for(int i = 0; i < N; i++) {
                arr[i] = Integer.parseInt(st.nextToken());
                sum += arr[i];
            }
            double avg = sum / N;
            double cnt = 0;
            for(int i = 0; i < N; i++) {
                if(arr[i] > avg) {
                    cnt++;
                }
            }

            System.out.printf("%.3f%%\n", (cnt/N)*100);

        }
    }
}